tests/models/xapi/test_lms.py

```diff
@@ -0,0 +1,260 @@
+    assert (
+        statement.object.definition.type == "http://adlnet.gov/expapi/activities/course"
+    )
+
+
+@custom_given(LMSUnregisteredCourse)
+def test_models_xapi_lms_unregistered_course_with_valid_statement(statement):
+    """Tests that a valid unregistered to a course statement has the expected
+    `verb`.`id` and `object`.`definition`.`type` property values.
+    """
+
+    assert statement.verb.id == "http://id.tincanapi.com/verb/unregistered"
+    assert (
+        statement.object.definition.type == "http://adlnet.gov/expapi/activities/course"
+    )
+
+
+@custom_given(LMSAccessedPage)
+def test_models_xapi_lms_accessed_page_with_valid_statement(statement):
+    """Tests that a valid accessed a page statement has the expected `verb`.`id` and
+    `object`.`definition`.`type` property values.
+    """
+
+    assert statement.verb.id == "https://w3id.org/xapi/netc/verbs/accessed"
+    assert (
+        statement.object.definition.type
+        == "https://w3id.org/xapi/acrossx/activities/webpage"
+    )
+
+
+@custom_given(LMSAccessedFile)
+def test_models_xapi_lms_accessed_file_with_valid_statement(statement):
+    """Tests that a valid accessed a file statement has the expected `verb`.`id` and
+    `object`.`definition`.`type` property values.
+    """
+
+    assert statement.verb.id == "https://w3id.org/xapi/netc/verbs/accessed"
+    assert statement.object.definition.type == "http://activitystrea.ms/file"
+
+
+@custom_given(LMSUploadedFile)
+def test_models_xapi_lms_uploaded_file_with_valid_statement(statement):
+    """Tests that a valid uploaded a file statement has the expected `verb`.`id` and
+    `object`.`definition`.`type` property values.
+    """
+
+    assert statement.verb.id == "https://w3id.org/xapi/netc/verbs/uploaded"
+    assert statement.object.definition.type == "http://activitystrea.ms/file"
+
+
+@custom_given(LMSDownloadedFile)
+def test_models_xapi_lms_downloaded_file_with_valid_statement(statement):
+    """Tests that a valid downloaded a file statement has the expected `verb`.`id` and
+    `object`.`definition`.`type` property values.
+    """
+
+    assert statement.verb.id == "http://id.tincanapi.com/verb/downloaded"
+    assert statement.object.definition.type == "http://activitystrea.ms/file"
+
+
+@custom_given(LMSDownloadedVideo)
+def test_models_xapi_lms_downloaded_video_with_valid_statement(statement):
+    """Tests that a valid downloaded a video statement has the expected `verb`.`id` and
+    `object`.`definition`.`type` property values.
+    """
+
+    assert statement.verb.id == "http://id.tincanapi.com/verb/downloaded"
+    assert (
+        statement.object.definition.type
+        == "https://w3id.org/xapi/video/activity-type/video"
+    )
+
+
+@custom_given(LMSUploadedVideo)
+def test_models_xapi_lms_uploaded_video_with_valid_statement(statement):
+    """Tests that a valid uploaded a video statement has the expected `verb`.`id` and
+    `object`.`definition`.`type` property values.
```

Comment:
```suggestion
    """Tests that a valid uploaded a video statement has the expected `verb.id` and
    `object.definition.type` property values.
```
Perhaps a bit more readable ? Or is there a reason for this syntax ? (Same goes for the other docstrings)

Reply:
It's a pattern we have chosen when refactoring the model docstrings